convert.py

```diff
@@ -0,0 +1,1053 @@
+from sentencepiece import SentencePieceProcessor # type: ignore
+import json, struct, re, zipfile, pickle, itertools, sys, enum, concurrent.futures
+import argparse, math, io, functools, dataclasses, mmap, signal, faulthandler, copy
+from pathlib import Path
+import numpy as np
+from dataclasses import dataclass
+from typing import Optional, Callable, Type, Any, Iterable, IO, Sequence, Union, TypeVar, Literal
+from abc import abstractmethod, ABCMeta
+
+faulthandler.register(signal.SIGUSR1)
```

Comment:
`faulthandler.register` does not exist on Windows, will throw AttributeError. ([link to doc](https://docs.python.org/3/library/faulthandler.html#dumping-the-traceback-on-a-user-signal))

Maybe we can reference this [PR](https://github.com/ipython/ipykernel/pull/108) in ipykernel for how to handle this, or just skip it?